File: logicaProgramacaoPROA/visualG-and-JS/FIT-FACCAT/VISUALG/JS/exercicio40/js/exercicio40.js
let precoUni,desconto,quantidade,total,totalPagar

nome= prompt("Qual o nome do produto: ")
quantidade= parseFloat(prompt("Quantos você comprou? "))
precoUni=parseFloat(prompt("Qual é o valor unitário?"))

total=quantidade*precoUni

if (quantidade <=5){
    desconto=total*(2/100)
}
else if (quantidade>5 && quantidade<=10){

    desconto=total*(3/100)
}

else {
    desconto=total*(5/100)
}

totalPagar=total-desconto
alert("O preço total do produto"+ nome+ "a pagar é R$"+ totalPagar)

// escreval("Qual o nome do produto?")
// leia(nome)
// escreval("Quantos você comprou?")
// leia(quantidade)
// escreval("Qual é o valor unitario?")
// leia(precoUni)
// total<-quantidade*precoUni
// se quantidade<=5 entao
// desconto<-total*(2/100)
// senao
// se (quantidade > 5) e (quantidade<=10) entao
// desconto<-total*(3/100)
// senao
// desconto<-total*(5/100)
// fimse
// fimse
// totalPagar<-total-desconto
// escreval("O preço total do produto",nome," a pagar é R$",totalPagar)

File: logicaProgramacaoPROA/JAVA/FACCAT/Exercicio40.java
package javaBasico;

import java.util.Scanner;

public class Exercicio40 {
	
	public static void main(String args[]) {
		
		// SCANNER
		Scanner sc = new Scanner(System.in);
		
		// VARIÁVEIS
		String nome;
		int quantidade;
		double preco, total, totalPagar;
		
		// CADASTRO DE PRODUTOSA
		System.out.println("Comprar produto \n"
				+ "\nInsira o nome do produto: ");
		nome = sc.next();
		System.out.println("Insira a quantidade adquirida: ");
		quantidade = sc.nextInt();
		System.out.println("Insira o preço unitário: ");
		preco = sc.nextDouble();
		
		// TOTAL DA COMPRA
		total = quantidade * preco;
		
		// DESCONTO DE COMPRA
		if (quantidade <= 5) {
			totalPagar = total - (total * 0.2);
			System.out.println("\nItem: " + nome);
			System.out.printf("O total a pagar é de R$ %.2f", totalPagar);
		}
		else if (quantidade > 5 && quantidade <= 10) {
			totalPagar = total - (total * 0.3);
			System.out.println("\nItem: " + nome);
			System.out.printf("O total a pagar é de R$ %.2f", totalPagar);
		}
		else {
			totalPagar = total - (total * 0.5);
			System.out.println("\nItem: " + nome);
			System.out.printf("O total a pagar é de R$ %.2f", totalPagar);
		}
		sc.close();
	}
}
